pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -336,11 +353,30 @@ func ConfigPath(ctx *types.SystemContext) string {
 	return systemRegistriesConfPath
 }
 
+// configDirPath returns the path to the system-wide registry configuration
+// directory.
+func configDirPath(ctx *types.SystemContext) string {
```

Comment:
Is it reasonable to decide on the `.conf` and the `.d` independently? I’m leaning towards deciding on the two as a unit (i.e., assuming there are no `SystemContext` overrides, look for per-user `.d` iff we are using a per-user `.conf`).

At least system-wide overrides for a per-user config would be surprising.  OTOH per-user overrides for a system-wide config might make sense.

Reply:
I think it makes sense to decide independently. Maybe a user wants to follow whatever the admin (or package) decides in `/etc/containers` but also wants to dump their stuff into `~/.config/containers/registries.d`.